src/nvim/fileio.c

```diff
@@ -2734,9 +2734,11 @@ buf_write(
 
     if (backup_copy) {
       char_u *wp;
+      const bool kNoPrependDot = false;
```

Comment:
`kNoPrependDot` is never re-assigned so I guess this is for documentation/readability. But why declare it in 2 different scopes?